File: 프로그래머스/2/42584. 주식가격/주식가격.py
from collections import deque

def solution(prices):
    answer = [];
    q = deque(prices);
    
    while q :
        stock = q.popleft();
        idx = 0;
        for i in q :
            idx += 1;
            if i < stock :
                break;
        answer.append(idx);
        
    return answer

File: 프로그래머스/2/42584. 주식가격/주식가격.js
function solution(prices) {
    var answer = [];
    
    for(let i=0;i<prices.length;i++) {
        let count = 0;
        for(let j=i+1;j<prices.length;j++) {
            if(prices[i] > prices[j]) {
                count += 1;
                break;
            }
            else count += 1;
        }
        answer.push(count);
    }
    return answer;
}